gerritbot/src/lib.rs

```diff
@@ -415,7 +415,7 @@ where
             .chain(std::iter::once(&change.owner))
             .filter(|user| user.is_human())
             .filter_map(|user| user.spark_email())
-            .filter_map(move |email| self.state.find_user_by_email(email))
+            .filter_map(move |email| self.state.find_user(email))
```

Comment:
Apparently now you always return something, so you might want to remove the option here.

Reply:
Surprisingly, since both `find_user_by_email` and `find_user` return an option. Will look into the clippy warnings

Reply:
Looks like this warning was already before there. Feel free to ignore.